src/main/java/com/sinthoras/visualprospecting/integration/voxelmap/IWaypointManagerReflection.java

```diff
@@ -0,0 +1,23 @@
+package com.sinthoras.visualprospecting.integration.voxelmap;
+
+import java.lang.reflect.Method;
+
+import com.thevoxelbox.voxelmap.interfaces.IWaypointManager;
+
+public class IWaypointManagerReflection {
+
+	private static Method getCurrentSubworldDescriptor;
+	
+	public static String getCurrentSubworldDescriptor(IWaypointManager obj, boolean arg) {
+		try {
+			if(getCurrentSubworldDescriptor == null) {
+				getCurrentSubworldDescriptor = IWaypointManager.class.getMethod("if", boolean.class);
```

Comment:
You can extract this line into the static initializer of the class like here: https://github.com/SinTh0r4s/HydroEnergy/blob/3fec6a526544acf05a78e4190d0b5da61b5a8939/src/main/java/com/sinthoras/hydroenergy/HEReflection.java#L13

Reply:
That is actually much cleaner, changed it.